### ISSUE TITLE
Can't load admin page on CDMX server

### ISSUE THREAD

**Issue by misaugstad** [ProjectSidewalk/SidewalkWebpage#2023]
If you try to go to the admin page on the CDMX server (either production or test) I get the following error:
```
onError Happened
Execution exception[[SlickException: Read NULL value (null) for ResultSet column Path s2._7]]
```

I think this is one of those errors that is really annoying to debug because the error message is not at all descriptive (even in the sbt console), and the only way to find out where the error is coming from is to comment out sections of the template HTML for the admin page until the error goes away (and then you know the error was in that commented out section).

**Comment by misaugstad** [comment]
Fixed. It was an error of mine when creating the database. The `street_edge` table has a `way_type` field that comes from OSM. We use that info to pick which types of streets to include in our road network (for example, we exclude freeways). Somehow I included roads in the database that have a null `way_type`.

Upon further investigation, these turned out to be "ways" that must have had a different way that the way type was marked because they are not traditional roads. These "ways" ended up being strips of runway at an airport or railroad tracks, so they weren't marked the way roads are marked.

To fix, I just set the `deleted` column to true in the database for "streets" with a null `way_type` column.

**Comment by misaugstad** [comment]
So this happened again, documented in #2110. It happened because I added more neighborhoods to the city and didn't remember to filter out those streets with null `way_type`. I realized that there's no reason to keep those in the database anyway; they should have never been there in the first place. So I deleted all of those streets from the database (except for the 3 that already had audits). Should prevent this from ever happening again.
